Virtual Librarian/VirtualLibrarian.Domain/User.cs

```diff
@@ -21,15 +21,16 @@ public enum Faculty
             CGF
         }
         public string Name { get; }
-        public string Surname { get; }
+        //public string Surname { get; }
         public Faculty CurrentFaculty { get; set; }
         public int Id { get; }
         public List<BookCopy> takenBooks = new List<BookCopy>();
-        public User(int id, string name, string surname, Faculty faculty = Faculty.MIF)
+        //public User(int id, string name, string surname, Faculty faculty = Faculty.MIF)
```

Comment:
please make these changes to domain.models.User, because this one is not used any more